lib/grandpa/message_handler_test.go

```diff
@@ -1055,3 +1063,152 @@ func signFakeFullVote(
 
 	return sig
 }
+
+func TestService_VerifyBlockJustification(t *testing.T) {
+	validJustification := "0x05000000000000008306f3c4dda0211e863e52c8bc15a3251e5ba38c7dd2f241f896653665e5e7e0080000" +
```

Comment:
Maybe better to generate this if possible? If not feasible within unit test, maybe comment where you got it?

Reply:
Good point, I've refactored this to use `buildTestJustification` to generate this, it's clearer now what the test is testing.